drivers/scale_service.go

```diff
@@ -44,6 +44,10 @@ func (s *ScaleServiceDriver) ValidatePayload(conf interface{}, apiClient client.
 		return http.StatusBadRequest, fmt.Errorf("Maximum scale not provided/invalid")
 	}
 
+	if config.Min >= config.Max {
+		return http.StatusBadRequest, fmt.Errorf("Min cannot be greater than max")
```

Comment:
Error statement is no longer accurate due to the >=. Change it to 'Max must be greater than min'.